src/index.html

```diff
@@ -127,6 +128,67 @@ <h2 class="chart-period mb text-center">
           </small>
         </p>
       </footer>
+
+      <div id="settings-panel" class="settings hidden">
+        <button id="close-settings" class="close-button">&times;</button>
+        <h2>Customize Panel</h2>
+        <div class="settings-container">
+          <label>Theme:</label>
+          <div class="toggle-container">
+            <div id="theme-toggle" class="toggle">
+              <div class="toggle-option active" data-theme="light">Light</div>
+              <div class="toggle-option" data-theme="dark">Dark</div>
+            </div>
+          </div>
+        </div>
+        <div class="settings-container">
+          <label for="token-select">Token:</label>
+          <!-- Disabled until we have more token support. -->
+          <select id="token-select" disabled>
+            <option value="bitcoin">Bitcoin</option>
+            <option value="ethereum">Ethereum</option>
+          </select>
+        </div>
+        <div class="settings-container">
+          <label for="clock-format">Clock Format:</label>
+          <select id="clock-format">
+            <option value="24h">24 Hour</option>
+            <option value="12h">12 Hour</option>
+          </select>
+        </div>
+        <div class="settings-container">
+          <label for="clock-up-down">UP and DOWN colors:</label>
```

Comment:
I'd love to see option to "reset" to the default colors (for users that tend to make many changes).

Reply:
The "reset to default colors" option is ready! Let me know if you’d like it tweaked.